src/components/Badge/index.tsx

```diff
@@ -46,7 +46,12 @@ const BaseBadge = styled(Txt.span)`
 	line-height: 1;
 `;
 
-const Badge = ({ children, className, shade }: ThemedBadgeProps) => {
+const Badge = ({
```

Comment:
This refactor should be in a separate commit, or even better, a separate PR.

Reply:
Sure, I can move it into a separate commit.